internal/policies/mount/adsys_mount/run_test.go

```diff
@@ -0,0 +1,36 @@
+// Package adsysmount_test runs rust tests from a go test global command
+package adsysmount_test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/ubuntu/adsys/internal/testutils"
+)
+
+func TestRust(t *testing.T) {
+	t.Parallel()
+
+	// properly inform Go about which files to use for cache by reading input files.
+	testutils.MarkRustFilesForTestCache(t)
+
+	cmd := exec.Command("cargo", "test")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		t.Fail()
```

Comment:
Shouldn't we log the error here as well (even though we are already setting the cmd stds to the os ones)?

Reply:
No really, I tried it with failing tests and then, the error message makes sense: you see that this test fails, and you get the old stdout/stderr printed above, which is what makes sense and is in line with pure Go tests.